src/index.js

```diff
@@ -51,13 +52,13 @@ const checkSignInStatus = () => {
   // = $('#username').val().split('r')[1]
   if ($('#username').val().split('r')[0] === 'custome') {
     userIdNumber = parseInt($('#username').val().split('r')[1])
-    instanciateCustomer(userIdNumber);
     instanciateHotel();
+    instanciateCustomer(userIdNumber);
     domUpdates.displayCustomerWelcomeScreen(signedInUser);
     domUpdates.populatePastFutureReservations(signedInUser);
     hideOrShowElement('show', '.past-future-container, .book-a-cabin-container, .log-out-button, .user-validation');
     hideOrShowElement('hide', '.landing-container');
-    $(".date-input").attr("min", getTodaysDate().split('/').join('-') );
+    $(".date-input").attr("min", getTodaysDate().split('/').join('-'));
   } else if ($('#username').val() === 'manager' && $('#password').val() === 'overlook19') {
     instanciateHotel();
     instantiateManager();
```

Comment:
Watch out on how you are spelling instantiate!  Looks like there is a discrepancy between instanciateHotel vs instantiateManager